src/wallet/wallet2.h

```diff
@@ -638,6 +641,9 @@ namespace tools
     void set_tx_note(const crypto::hash &txid, const std::string &note);
     std::string get_tx_note(const crypto::hash &txid) const;
 
+    void set_wallet_note(const std::string &note);
+    std::string get_wallet_note();
```

Comment:
const

Reply:
Sorry, not sure where additional `const` is needed/advisable. The corresponding methods for tx notes where I looked for guidance don't have any more `const` than this.

Reply:
Replace:
std::string get_wallet_note();
with:
std::string get_wallet_note() const;

And likewise in the implementation of the function. It means this function is not allowed to change the wallet2 object (it only reads).

Reply:
Ah, of course, I completely overlooked the const at the end of the method declaration...